webmunk-ext/src/background/worker.js

```diff
@@ -9,6 +9,30 @@ import "@webmunk/cookies-scrapper-module/worker";
 const appMgr =  {
   initialize:function(){
     messenger?.addReceiver('appMgr', this);
+    this.isSurveyCompletedAfterSurvey();
+  },
+
+  isSurveyCompletedAfterSurvey() {
+    const runtimeUrl = `chrome-extension://${chrome.runtime.id}/pages/survey-completed.html`;
+
+    chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
```

Comment:
move chrome.tabs.onUpdated.addListener to initialize function and rename method to survayCompleteListener